packages/oui-navbar/src/group/navbar-group.controller.js

```diff
@@ -1,11 +1,57 @@
 export default class {
-    constructor ($attrs, $element, ouiNavbarConfiguration, NavbarGroupService) {
+    constructor ($attrs, $element, ouiNavbarConfiguration, KEYBOARD_KEYS) {
         "ngInject";
 
         this.$attrs = $attrs;
         this.$element = $element;
         this.config = ouiNavbarConfiguration;
-        this.navbarGroupService = NavbarGroupService;
+
+        this.KEYBOARD_KEYS = KEYBOARD_KEYS;
+    }
+
+    bindGroup (groupName) {
+        const keys = {};
+        const keysRegex = new RegExp([
+            this.KEYBOARD_KEYS.TAB,
+            this.KEYBOARD_KEYS.SHIFT
+        ].join("|"));
```

Comment:
Yes, it could be `const keysRegex = new RegExp(`${this.KEYBOARD_KEYS.TAB}|${this.KEYBOARD_KEYS.SHIFT}`);` but i find this formulation more readable (and more elegant) :)